src/orion/algo/nevergrad/nevergradoptimizer.py

```diff
@@ -25,8 +105,14 @@ class NevergradOptimizer(BaseAlgorithm):
     requires_dist = None
     requires_shape = None
 
-    def __init__(self, space, seed=None):
-        super(NevergradOptimizer, self).__init__(space, seed=seed)
+    def __init__(self, space, seed=None, budget=100, num_workers=10):
+        param = to_ng_space(space)
+        self.algo = ng.optimizers.NGOpt(
+            parametrization=param, budget=budget, num_workers=num_workers
+        )
+        self.algo.enable_pickling()
+        self._trial_mapping = {}
+        super().__init__(space, seed=seed)
```

Comment:
```suggestion
        super().__init__(space, seed=seed, budget=budget, num_workers=num_workers)
```
The arguments need to be passed to the parent's constructor to be included into the configuration of the algorithm automatically.